dm_control/locomotion/walkers/assets/dog_v2/build_dog.py

```diff
@@ -17,44 +17,75 @@
 
 import os
 
-from absl import app
-from absl import flags
-from dm_control import mjcf
-from dm_control.locomotion.walkers.assets.dog_v2 import add_torque_actuators
-from dm_control.locomotion.walkers.assets.dog_v2 import build_back_legs
-from dm_control.locomotion.walkers.assets.dog_v2 import build_front_legs
-from dm_control.locomotion.walkers.assets.dog_v2 import build_neck
-from dm_control.locomotion.walkers.assets.dog_v2 import build_tail
-from dm_control.locomotion.walkers.assets.dog_v2 import build_torso
-from dm_control.locomotion.walkers.assets.dog_v2 import create_skin
-from lxml import etree
 import numpy as np
+from absl import app, flags
+from lxml import etree
 
+from dm_control import mjcf
+from dm_control.locomotion.walkers.assets.dog_v2 import add_markers as add_markers_func
```

Comment:
This style of import is against the Google Python Style (don't import individual functions): https://google.github.io/styleguide/pyguide.html#s2.2-imports

Just have:
```
from dm_control.locomotion.walkers.assets import dog_v2
```

and for all the individual functions you imported, just call dog_v2.add_muscles, etc.